Calendar/Parse Handlers/LogoutHandler.m

```diff
@@ -0,0 +1,28 @@
+//
+//  LogoutHandler.m
+//  InstagramClone
+//
+//  Created by Angelina Zhang on 6/27/22.
+//
+
+#import "LogoutHandler.h"
+#import "SceneDelegate.h"
+#import "LoginViewController.h"
+#import <Parse/Parse.h>
+
+@implementation LogoutHandler
+
+- (void)logout {
+    [PFUser logOutInBackgroundWithBlock:^(NSError * _Nullable error) {
+        if (error) {
+            [self.delegate failedLogout];
+        }
+    }];
+    SceneDelegate *sceneDelegate = (SceneDelegate * ) UIApplication.sharedApplication.connectedScenes.allObjects.firstObject.delegate;
+    
+    UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
+    LoginViewController *loginViewController = [storyboard instantiateViewControllerWithIdentifier:@"LoginViewController"];
+    sceneDelegate.window.rootViewController = loginViewController;
+}
```

Comment:
I would do this in the completion. What happens if logout fails? You're stranded at the beginning of the app. It's a weird user experience.